### PR TITLE
enhancement/geometry-demos

### PR DESCRIPTION
**Pull request by magdalena-gut**
Updated `map.data.geometry` API with the new demo.

### CODE REVIEW

File: samples/maps/series/geometry-types/demo.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                                [-63, -20],
+                                [-56, -14],
+                                [-45, -3],
+                                [-63, 9]
+                            ]
+                        ]
+                    }
+                }]
+            },
+            {
```
Comment (hubertkozik) [inline samples/maps/series/geometry-types/demo.js:99]: Unnecessary enter.

File: samples/maps/series/geometry-types/demo.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                                    [26, -32],
+                                    [28, -21],
+                                    [13, -17]
+                                ]
+                            ]
+                        ]
+                    }
+                }]
+            }
+            ]
```
Comment (hubertkozik) [inline samples/maps/series/geometry-types/demo.js:132]: Unnecessary enter.

**Review by hubertkozik — APPROVED** [review]
Great! Just formatting comments below.

File: samples/maps/series/geometry-types/demo.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+<script src="https://code.highcharts.com/maps/highmaps.js"></script>
+<script src="https://code.highcharts.com/maps/modules/data.js"></script>
+<script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/maps/modules/offline-exporting.js"></script>
```
Comment (KacperMadej) [inline samples/maps/series/geometry-types/demo.html:5]: Why no a11y module if exporting & offline-exporting are added?

File: ts/Series/Map/MapSeries.ts [hunk 4]
```diff
@@ -1551,6 +1551,8 @@ export default MapSeries;
  *
  * @sample maps/series/data-geometry/
  *         Geometry defined in data
```
Comment (KacperMadej) [inline ts/Series/Map/MapSeries.ts:1553]: Maybe renaming the original demo to its title "Map point and line geometry" could make it easier to know what is the difference between those demos just from the name in the API docs?

**Review by KacperMadej — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**